Source/WebKit/UIProcess/API/Cocoa/WKWebpagePreferencesPrivate.h

```diff
@@ -74,6 +74,12 @@ typedef NS_OPTIONS(NSUInteger, _WKWebsiteColorSchemePreference) {
     _WKWebsiteColorSchemePreferenceDark,
 } WK_API_AVAILABLE(macos(13.0), ios(16.0));
 
+// Allow controlling a per-page network connection integrity policy.
+typedef NS_OPTIONS(NSUInteger, _WKWebsiteNetworkConnectionIntegrityPolicy) {
+    _WKWebsiteNetworkConnectionIntegrityPolicyEnabled,
```

Comment:
Nit - since these are enum flags, let's explicitly make their values `= 1 << 0` and `= 1 << 1`.